pruneArcBuild.js

```diff
@@ -0,0 +1,27 @@
+const glob = require("glob");
+var fs = require("fs");
+
+const files = glob.sync("./node_modules/@daostack/arc/build/contracts/*.json", {
+  nodir: true
+});
+
+console.log(`Starts pruning Arc JSON files`);
```

Comment:
Suggestion: could use `ora` to make the prints more pretty.

Reply:
Will add

Reply:
Fixed